### ISSUE TITLE
[macOS] [3.2] Build fails - 'free' undeclared

### ISSUE THREAD

**Issue by asmaloney** [godotengine/godot#46104]
**Godot version:**

3.2 branch 30c69c296b66c353fe49a56b91d91b0207689114

**OS/device including version:**

macOS 10.14.6

```
$ clang --version
Apple clang version 11.0.0 (clang-1100.0.33.17)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
```

scons command: `scons target=debug dev=yes tools=yes --jobs=$(sysctl -n hw.logicalcpu)`

**Issue description:**

Compiler error:

```
godot3/modules/tinyexr/image_saver_tinyexr.cpp:279: error: use of undeclared identifier 'free'; did you mean 'DefaultAllocator::free'?
modules/tinyexr/image_saver_tinyexr.cpp:279:3: error: use of undeclared identifier 'free'; did you mean 'DefaultAllocator::free'?
                free(mem);
                ^~~~
                DefaultAllocator::free
./core/os/memory.h:66:29: note: 'DefaultAllocator::free' declared here
        _FORCE_INLINE_ static void free(void *p_ptr) { Memory::free_static(p_ptr, false); }
                                   ^
```

Introduced in [ Backport EXR compression support from master ](https://github.com/godotengine/godot/commit/7241139356e78e275e5f11c644e55e1c1963d021) (git hash 77438f7a453beac5d4a31c5690210a41e9187cec)

**Comment by asmaloney** [comment]
I'm surprised this wasn't caught by CI. Any idea why?

**Comment by akien-mga** [comment]
Fixed by #46115.

**Comment by akien-mga** [comment]
> I'm surprised this wasn't caught by CI. Any idea why?

Probably a different Xcode SDK version used between the GitHub Actions CI and what you have locally, and they might have removed this include from a Xcode SDK header which would previously include it. I guess?
